### PR TITLE
Bower support

### PR DESCRIPTION
**Pull request by sethkinast**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  "authors": [
+    "Veena Basavaraj <veena.braj@gmail.com>",
+    "Yevgeniy Brikman <jbrikman@linkedin.com>",
+    "Sarah Clatterbuck <sclatter@linkedin.com>",
+    "Jimmy Chan <jchan@linkedin.com>",
+    "Steven Foote <sfoote@linkedin.com>",
+    "Johnathan Leppert <jleppert@linkedin.com>",
+    "Prayrit Prash Jain <prjain@linkedin.com>",
+    "Seth Kinast <skinast@linkedin.com>",
+    "Jairo de Morais <jairodemorais@gmail.com>",
```
Comment (jimmyhchan) [inline bower.json]: at-linkedin    that might be his gmail though :)

File: package.json [hunk 2]
```diff
@@ -8,38 +8,18 @@
...
+    "Veena Basavaraj <veena.braj@gmail.com>",
+    "Yevgeniy Brikman <jbrikman@linkedin.com>",
+    "Tom Carchrae",
+    "Sarah Clatterbuck <sclatter@linkedin.com>",
+    "Jimmy Chan <jchan@linkedin.com>",
+    "Steven Foote <sfoote@linkedin.com>",
+    "Johnathan Leppert <jleppert@linkedin.com>",
+    "Prayrit Prash Jain <prjain@linkedin.com>",
+    "Seth Kinast <skinast@linkedin.com>",
+    "Jairo de Morais <jairodemorais@gmail.com>",
```
Comment (jimmyhchan) [inline package.json]: ditto

**Comment by jimmyhchan** [comment]
looks good :+1:

File: bower.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "dustjs-helpers",
+  "version": "1.6.0",
+  "homepage": "https://github.com/linkedin/dustjs-helpers",
+  "authors": [
+    "Veena Basavaraj <vybs@users.noreply.github.com>",
+    "Yevgeniy Brikman <jbrikman@linkedin.com>",
```
Comment (smfoote) [inline bower.json]: Should we update this one?

Comment (sethkinast) [inline bower.json]: gottem

**Pull request merged**